lang/programming.talon

```diff
@@ -34,6 +34,12 @@ state include local: user.code_include_local()
 state type deaf: user.code_type_definition()
 state type deaf struct: user.code_typedef_struct()
 state (no | nil | null): user.code_null()
+state break: user.code_break()
+state next: user.code_next()
+state true: user.code_true()
+state false: user.code_false()
+state na: user.code_na()
```

Comment:
`state na` I'm a little on the fence on - might be better as an R-specific think? From a cursory search, I don't think there's an analog in thing in many languages (python, C#, C++), though I could be very wrong!

Reply:
I wasn't sure either, I will change it to an R specific command.